proto/gen/einride/iam/example/v1/freight_service.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
+// 	protoc-gen-go v1.27.1
```

Comment:
There are a number of generated files commited with this change, but I see no version bump of `protoc-gen-go`.

Reply:
The dependabot PRs seem to pass the build without getting caught by the git diff check here - need to look into this.